docs/guides/migrations/guide.rst

```diff
@@ -0,0 +1,1298 @@
+data, or add some data to a migration that you have just created before 
+you apply it.
+
+EdgeDB assumes by default that a migration involves a change to your 
+schema, so it won't create a migration for you if it doesn't see a 
+schema change:
+
+.. code-block:: bash
+
+    $ edgedb migration create
+    No schema changes detected.
+
+So how do you create a migration with only data? To do this, just 
+add ``--allow-empty`` to the command:
+
+.. code-block:: bash
+
+    $ edgedb migration create --allow-empty
+    Created myproject/dbschema/migrations/00002.edgeql,
+    id: m1xseswmheqzxutr55cu66ko4oracannpddujg7gkna2zsjpqm2g3a
+
+You will now see an empty migration in which you can enter some queries. 
+It will look something like this:
+
+.. code-block::
+
+    CREATE MIGRATION m1xseswmheqzxutr55cu66ko4oracannpddujg7gkna2zsjpqm2g3a
+        ONTO m1n5lfw7n74626cverbjwdhcafnhmbezjhwec2rbt46gh3ztoo7mqa
+    {
+    };
+
+You can now add your queries inside the braces. Assuming a schema 
+with a simple ``User`` type, we could then add a bunch of queries 
+such as the following:
+
+.. code-block::
+
+    CREATE MIGRATION m1xseswmheqzxutr55cu66ko4oracannpddujg7gkna2zsjpqm2g3a
+        ONTO m1n5lfw7n74626cverbjwdhcafnhmbezjhwec2rbt46gh3ztoo7mqa
+    {
+        insert User { name := 'User 1'};
+        insert User { name := 'User 2'};
+        delete User filter .name = 'User 2';
+    };
+
+But when you type ``edgedb migrate``, the CLI will then complain that 
+the migration hash doesn't match what it is supposed to be. However, 
+it helpfully provides the reason: "Migration names are computed from 
+the hash of the migration contents".
+
+Fortunately, it also tells you exactly what the hash (the migration 
+name) will need to be and you can simply change it to that.
+
+.. code-block::
+
+    Error executing command: could not read migrations in 
+    myproject/dbschema/migrations:
+
+    could not read migration file myproject/dbschema/migrations/00002.edgeql:
+
+    Migration name should be:
+    m13g7j2tqu23yaffv6wkn2adp6hayp76su2qtg2lutdh3mmj5xyk6q, but
+    m1xseswmheqzxutr55cu66ko4oracannpddujg7gkna2zsjpqm2g3a found instead.
+
+
+    Migration names are computed from the hash of the migration contents.
+
+    To proceed you must fix the statement to read as:
+    CREATE MIGRATION m13g7j2tqu23yaffv6wkn2adp6hayp76su2qtg2lutdh3mmj5xyk6q 
+    ONTO ...
+    Alternatively, revert the changes to the file.
+
+You can also create a migration that combines schema changes and data. 
+This is even easier, since it doesn't even require appending ``allow-empty`` 
+to the command. Just do the following:
+
+- Change your schema,
+- Type ``edgedb migration create`` and respond to the CLI's questions,
+- Add your queries to the file (best done on the bottom after the 
+  DDL statements have changed the schema),
+- Type ``edgedb migrate`` and change the migration name to the suggested name,
+- Type ``edgedb migrate`` again.
```

Comment:
```suggestion
1. Change your schema
2. Type ``edgedb migration create`` and respond to the CLI's questions
3. Add your queries to the file (best done on the bottom after the 
   DDL statements have changed the schema)
4. Type ``edgedb migrate`` and change the migration name to the suggested name
5. Type ``edgedb migrate`` again
```
Changing to an ordered list since order is important here. Also ditching the commas since a series is implied by the list.